targets/csharp/source/PlayFabSDK/source/Uunit/UUnitIncrementalTestRunner.cs

```diff
@@ -88,6 +88,11 @@ public static TestTitleData VerifyTestTitleData()
 #endif
         }
 
+        public static void AddTestAssembly(Assembly assembly, string filter = null)
```

Comment:
I see a weird build failure from our tests: 

" Uunit\UUnitIncrementalTestRunner.cs(91,44): error CS0246: The type or namespace name 'Assembly' could not be found (are you missing a using directive or an assembly reference?) [C:\program files\jenkins\workspace\SDK_Work_CSharp\sdks\CSharpBetaSdk\PlayFabSDK\source\PlayFabSDK.csproj]"

Reply:
Good catch, thanks. I added the correct using when I was testing locally, but forgot to copy it over when I moved to SdkGenerator.